tools/CoqMakefile.in

```diff
@@ -801,7 +801,7 @@ $(addsuffix .d,$(MLPACKFILES)): %.mlpack.d: %.mlpack
 # projects. Note that extra options might be on the command line.
 VDFILE_FLAGS:=$(if @PROJECT_FILE@,-f @PROJECT_FILE@,) $(CMDLINE_COQLIBS) $(CMDLINE_VFILES)
 
-$(VDFILE): $(VFILES)
+$(VDFILE): $(if @PROJECT_FILE@,-f @PROJECT_FILE@,) $(VFILES)
```

Comment:
Seems wrong, I think you want it without the `-f`?  But in this case you don't even need the `if` statement, since if the variable is empty, you'll just get an extra space

Reply:
Yes, I made a confusion (and it was untested).

I used your suggestion that left a possible extra space since there is already a possible extra space in `VDFILE_FLAGS`